lxd/api_cluster.go

```diff
@@ -2655,13 +2656,31 @@ func evacuateClusterMember(d *Daemon, r *http.Request) response.Response {
 					val = 30
 				}
 
+				// Try to live-migrate instance depending on evacuation mode.
+				evacMode, present := inst.ExpandedConfig()["cluster.evacuate"]
```

Comment:
This feels a bit hackish, I think it's cleaner to extend the existing CanMigrate.

I've put some example code (untested) in the original issue.